File: 217-ContainsDuplicate/217-ContainsDuplicate.py
# Last updated: 6/12/2025, 8:28:12 PM
class Solution(object):
    def containsDuplicate(self, nums):
        lst=list()
        nums.sort()
        for i in range(0,len(nums)-1):
            if nums[i] ==nums[i+1]:
                return True
        return False
        """
        :type nums: List[int]
        :rtype: bool
        """
        

File: 217-ContainsDuplicate/217-ContainsDuplicate.cpp
// Last updated: 6/12/2025, 8:28:07 PM
class Solution {
public:
    bool containsDuplicate(vector<int>& nums) {
        set<int>check;
        for(int i=0;i<nums.size();i++)
        {
            if(check.find(nums[i])!=check.end())
            {
                return true;
            }
            check.insert(nums[i]);
        }
        return false;
    }
};